src/strategies/index.ts

```diff
@@ -154,6 +154,7 @@ import * as occStakeOf from './occ-stake-of';
 import * as hoprStaking from './hopr-staking';
 import * as hoprBridgedBalance from './hopr-bridged-balance';
 import * as lootCharacterGuilds from './loot-character-guilds';
+import * as anyanVaults from './anyan-staked-in-vaults';
```

Comment:
The folder name is different to strategy

Reply:
Sorry, I completely lost my concentration when I was working on it, corrected.